formatter_test.go

```diff
@@ -472,6 +472,61 @@ func TestFormatter_FailureDelta(t *testing.T) {
 	}
 }
 
+func TestFormatter_FailureErrors(t *testing.T) {
+	type caseStructure struct {
+		name     string
+		errors   []error
+		expected []string
+	}
+
+	cases := []caseStructure{
+		{
+			name:     "nil errors slice",
+			errors:   nil,
+			expected: nil,
```

Comment:
I found that the approach of `expected: []error{}` results in `[]error{[]error{}}` rather than `[]error{[]string(nil)}` so rather than using `expected: []string(nil)` works

Reply:
I was talking about changing DefaultFormatter implementation, so that would return empty slice instead of nil slice.

(yep, it's []string, not []error, my bad)

I've pushed commit to this branch.